resharper/resharper-unity/src/CSharp/Daemon/Stages/Highlightings/UnityHighlightingAttributeIds.cs

```diff
@@ -1,7 +1,10 @@
 ﻿using JetBrains.ReSharper.Feature.Services.Daemon;
 using JetBrains.ReSharper.Plugins.Unity.CSharp.Daemon.Stages.Highlightings;
+using JetBrains.ReSharper.Plugins.Unity.Feature.HighlightingEye;
 
-[assembly: RegisterConfigurableHighlightingsGroup(UnityHighlightingGroupIds.Unity, "Unity")]
+[assembly: RegisterConfigurableHighlightingsGroup(UnityHighlightingGroupIds.Unity, "Unity"),
+           RegisterConfigurableHighlightingsGroup(UnityHighlightingGroupIds.UnityPerformance, "Unity Performance Inspections", HighlightingEyeGroupKind.UnityPerformanceKind
```

Comment:
Yes, but that group is not new and was presented in 18.3

Reply:
I missed that. Do we need it to be a separate group? It would be nicer if they all lived under a single group. 

There's also an issue that the dialog isn't sorted alphabetically, which would at least put them close together - [RIDER-38239](https://youtrack.jetbrains.com/issue/RIDER-38239)

Reply:
I do not want to change right now, because it is used in our new hector to disable all performance highlightings in one click